book/http.md

```diff
@@ -657,7 +656,8 @@ followed by the tag name (and no attributes).
 
 So, to create our very, very simple web browser, let's take the page
 HTML and print all the text, but not the tags, in it.[^python2] I'll do
-this in a new function, `show`:
+this in a new function, `show`:^[Note that this is a global function and
+not really part of the browser proper.]
```

Comment:
I like the idea of a footnote but the exact text is odd. It’s part of the browser (in fact, it’s a proto-parser class) but maybe you meant not part of the URL class is what you meant?